lib/fog/azurerm/docs/storage.md

```diff
@@ -175,110 +176,145 @@ List all the storage containers in the current storage accounts.
 
 ```ruby
 azure_storage_service.directories.each do |directory|
-  puts "#{directory.name}"
+  puts directory.key
 end
 ```
 
-## Get the access control list of the storage container
+## Get the access control level of the storage container
 
 Get the permissions for the specified container. The permissions indicate whether container data may be accessed publicly.
 
 ```ruby
-directory = azure_storage_service.directories.get('<container name>')
-access_control_list = directory.get_access_control_list('<container name>')
-puts "#{access_control_list.inspect}"
+directory = azure_storage_service.directories.get('<Container Name>', max_results: 1)
+puts directory.acl
+```
+
+## Set the access control level of the storage container
+
+Set the permissions for the specified container. The permissions indicate whether container data may be accessed publicly.
+
+```ruby
+directory = azure_storage_service.directories.get('<Container Name>', max_results: 1)
+directory.acl = 'container'
```

Comment:
Whats the meaning of 'container' in ACL terminology? Have we defined it somewhere?